src/components/Portfoliodetails.tsx

```diff
@@ -32,8 +33,7 @@ const Portfoliodetails = (props: any) => {
   
 
     const getData = async() => {
-        //http://3.236.213.150:8081/portfolios/6
-        axios.get(`http://3.236.213.150:8081/portfolios/${id}`)
+        axios.get(url + `/portfolios/${id}`)
```

Comment:
@ddinh please store `url + "/portfolios"` to a variable and pass here to minimize code duplication throughout this file